### PR TITLE
Move sentry over to the new setup

### PR DESCRIPTION
**Pull request by ryelo**

### CODE REVIEW

File: src/js/sentry.js [hunk 1]
```diff
@@ -1,44 +1,95 @@
...
+// Actually initialize sentry with the group's api key
 function initSentry() {
-    let betaCheck = (window.location.pathname.split('/')[1] === 'beta' ? ' Beta' : '');
 
+    const appDetails = getAppDetails();
+
+    let API_KEY;
+    switch (appDetails.app.group) {
+        case 'insights':
+            API_KEY = 'https://8b6372cad9604745ae3606bc4adc0060@sentry.io/1484024';
```
[the remark this answers is not in the document]
Comment (ryelo) [reply src/js/sentry.js]: I mean there's probably a better way we could do this, but my response follows the same one here: https://github.com/RedHatInsights/insights-chrome/pull/309#discussion_r288604856

File: src/js/chrome.js [hunk 2]
```diff
@@ -9,7 +9,10 @@ const libjwt = auth();
 function noop () {}
 
 libjwt.initPromise.then(() => {
-    libjwt.jwt.getUserInfo().then(analytics && sentry).catch(noop);
+    libjwt.jwt.getUserInfo().then((...data) => {
```
[the remark this answers is not in the document]
Comment (karelhala) [reply src/js/chrome.js]: you have to have the first spread, to collect all attributes that are needed to be passed to analytics and sentry.

File: src/js/sentry.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (karelhala) [reply src/js/sentry.js]: We could store these keys in ENV variables or .env file and read them from there.

**Comment by ryelo** [comment]
@kruai Is this good for now? I think it'd be smart to open another PR later to fix all the api_keys. But, I think that's out of scope of this PR.

**Pull request merged**